### DISCUSSION TITLE
Sort and organize collections

### DISCUSSION THREAD

**Discussion by sylvaindeloux** [directus/directus#5093] [Feature Requests]
Hi

It would be interesting to allow sort on collections, so the final app user will see organized elements in the sidebar, instead of an alphabetical order.
Maybe add the possibility to split collections into folders ?

Thanks :)

**Comment by benhaynes** [comment]
You can do this now with the navigation override within a role... but I agree that a global option with a more intuitive interface would be nice.

**Reply by sylvaindeloux** [reply]
Oh that's nice, I didn't saw this option.
That's exactly what I'm asking, thanks :)
Useful if you don't have many groups

**Reply by martin-braun** [reply]
@benhaynes It's not very intuitive and I think it absolutely would make sense to have a global option as well for a base configuration.

**Comment by lukas-schaetzle** [comment]
I think in the long term it would be quite useful to have folder support for collections. For example, if you think of the many to any interface, you need a collection for every possible component. That can sum up to quite a number. I know, that we currently can hide collections, too. But the data models overview still lists all collections in a flat list which gets a bit crowded over time. I imagine that I can create a seperate folder for all builder components, for all page collections, helper collections (like tags, categories, ...) etc.That would improve UX for editors and admins.

**Reply by benhaynes** [reply]
I agree that it might be nice to have more/better organization for collections. To make this really useful, we might want to make sure this exists at the global level to avoid repeated configuration for each role.

**Reply by martin-braun** [reply]
So if I understand it correctly, we have one level folder support by using navigation collection grouping, but your point is to add folder support to enable multi level nesting of groups? @Tummerhore I agree with you. @benhaynes It might be possible to add groups to groups, thus infinite nesting would be possible. I also think it would be nice if you could add one collection/group to multiple groups to make your collection overview trivial for any end-user who uses it.

**Reply by benhaynes** [reply]
Hmm, I believe you can use Collection or Folders to deeply next collections now... so this organization should already be possible. Or am I misunderstanding the question?

![CleanShot 2022-08-04 at 09 31 46](https://user-images.githubusercontent.com/522079/182859718-ff17378c-f049-4d44-9674-fab50d17e454.png)

**Reply by martin-braun** [reply]
@benhaynes I can see that, but I think it would not allow redundancy of collections, it's not a huge limitation as of now and I will get back to this discussion when I have a valid use-case that is limited in this feature. Thank you.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
